CHANGELOG.md

```diff
@@ -48,12 +48,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added CodeQL analysis to esp32, stm32, pico, and wasm workflows
 - Added Function.ex and Protocol.ex improving Elixir 1.18 support
 - Added WiFi support for ESP32P4 via esp-wifi-external for build with ESP-IDF v5.4 and later
+- Added `system_info/1` key `atomvm_free_heap_size` support for STM32
+- Added `system_info/1` key `atomvm_minimum_free_size` support for STM32
```

Comment:
Let's rephrase this:
Added to `erlang:system_info/1` options for ... memory on STM32 or something like that.
There is no need to keep 2 different lines here.